platform/ios/src/MGLMapView.mm

```diff
@@ -1167,7 +1167,8 @@ - (void)touchesBegan:(__unused NS_SET_OF(UITouch *) *)touches withEvent:(__unuse
 }
 
 - (void)notifyGestureDidBegin {
-    [self notifyMapChange:mbgl::MapChangeRegionWillChange];
+    bool animated = false;
```

Comment:
👍 As a rule of thumb, in Objective-C++ code, if you’re inside an Objective-C class or category (look for `@implementation` and `@end`), use Objective-C types; if you’re inside a C++ `class` or `struct`, use C++ types. There are some exceptions: for example, I find it useful to use `static_cast` over C-style casts throughout Objective-C++ code. And `std::vector` can often be faster than NSMutableArray. [This article](https://pspdfkit.com/blog/2016/swifty-objective-c/) suggests some more ways C++ can improve Objective-C code, but a lot of it is a matter of preference.